### ISSUE TITLE
[Exoplayer] Source error

### ISSUE THREAD

**Issue by Matssa56** [jellyfin/jellyfin-android#1217]
### Describe the bug

Hi, I keep getting a "Source error" on the integrated player on my Pixel 6 running Android 14.
The movie itself is Ratatouille, below are the movie informations.
```
Ratatouille [Multi 4k]
Conteneur: mkv
Chemin: /data/mount/FILMS/Ratatouille [Multi 4k].mkv
Taille: 2949 MB

Vidéo
Titre: 4K HEVC HDR
Codec: HEVC
Profil: Main 10
Niveau: 153
Résolution: 3840x1604
Ratio d'aspect original: 2.40:1
Entrelacé: No
Images par seconde: 23.976025
Débit: 3711 kbps
Profondeur en Bit: 10 bit
Plage vidéo: HDR
Video range type: HDR10
Espace couleur: bt2020nc
Transfert couleur: smpte2084
Couleurs primaires: bt2020
Format de pixel: yuv420p10le
Images de référence: 1

Audio
Titre: FR VFF : AC3 5.1 - Fre - Dolby Digital - Default
Langue: fre
Codec: AC3
Répartition: 5.1
Chaînes: 6 ch
Débit: 640 kbps
Débit échantillon: 48000 Hz
Défaut: Yes
Forcé: No
Externe: No

Audio
Titre: ENG VO : AC3 5.1 - Dolby Digital
Langue: eng
Codec: AC3
Répartition: 5.1
Chaînes: 6 ch
Débit: 640 kbps
Débit échantillon: 48000 Hz
Défaut: No
Forcé: No
Externe: No

Sous-titre
Titre: FR Full : SRT - Fre - SUBRIP
Langue: fre
Codec: SUBRIP
Défaut: No
Forcé: No
Externe: No

Sous-titre
Titre: ENG Full : SRT - SUBRIP
Langue: eng
Codec: SUBRIP
Défaut: No
Forcé: No
Externe: No
```

### Logs

```shell
I don't know how to get the client logs, but here are the server logs (no errors found) :

023-11-10 15:52:06.032 +01:00] [INF] [21] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "<redacted>" request
[2023-11-10 15:52:23.889 +01:00] [INF] [20] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2023-11-10 15:52:35.890 +01:00] [INF] [20] Emby.Server.Implementations.Session.SessionWebSocketListener: Lost 1 WebSockets.
[2023-11-10 15:52:59.897 +01:00] [INF] [30] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2023-11-10 15:53:11.902 +01:00] [INF] [23] Emby.Server.Implementations.Session.SessionWebSocketListener: Lost 1 WebSockets.
[2023-11-10 15:54:00.912 +01:00] [INF] [21] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "<redacted>". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: False EnableAudioPlaybackTranscoding: False
[2023-11-10 15:54:00.913 +01:00] [INF] [21] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Jellyfin Android", Path="/data/mount/FILMS/Ratatouille [Multi 4k].mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/8b1a9299-2523-8de8-e892-7b1417a3ff74/stream.mkv?MediaSourceId=8b1a929925238de8e8927b1417a3ff74&Static=true&VideoCodec=hevc&AudioCodec=ac3&AudioStreamIndex=2&SubtitleStreamIndex=3&api_key=<token>&SubtitleMethod=Embed&Tag=bc2ca9757e3127cb749e5217c21d1502"
[2023-11-10 15:54:07.637 +01:00] [INF] [33] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Android" "2.5.3" playing "Ratatouille". Stopped at "0" ms
[2023-11-10 15:54:09.682 +01:00] [INF] [27] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "<redacted>". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: False EnableAudioPlaybackTranscoding: False
[2023-11-10 15:54:09.683 +01:00] [INF] [27] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Jellyfin Android", Path="/data/mount/FILMS/Ratatouille [Multi 4k].mkv", AudioStreamIndex=null, SubtitleStreamIndex=null ) => ( PlayMethod=DirectPlay, TranscodeReason=0 ) "media:/videos/8b1a9299-2523-8de8-e892-7b1417a3ff74/stream.mkv?MediaSourceId=8b1a929925238de8e8927b1417a3ff74&Static=true&VideoCodec=hevc&AudioCodec=ac3&AudioStreamIndex=2&SubtitleStreamIndex=3&api_key=<token>&SubtitleMethod=Embed&Tag=bc2ca9757e3127cb749e5217c21d1502"
[2023-11-10 15:54:11.679 +01:00] [INF] [27] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Android" "2.5.3" playing "Ratatouille". Stopped at "0" ms
```

### Application version

2.5.3

### Where did you install the app from?

Google Play

### Device information

Pixel 6

### Android version

Android 14

### Jellyfin server version

10.8.12

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [X] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

**Comment by solidsnake1298** [comment]
Can you also provide the server specs and OS?

**Comment by Matssa56** [comment]
Hi, sorry I thought that I added it. I was running Debian 12 with Docker. In terms of spec, it was under Proxmox on a NUC running on AMD Ryzen 7 7735HS and AMD Radeon 680M.

However I changed back to Plex after these issues and I also changed my NUC to an Intel Core i7-13620H with integrated graphics since apparently it works better for transcoding than AMD, even with a graphics card that seems higher on paper (thanks QuickSync).

I did not test this again since the NUC change though.
